README.md

```diff
@@ -46,28 +46,28 @@ The SpanId is 64-bit in length and indicates the position of the current operati
 
 The ParentSpanId is 64-bit in length and indicates the position of the parent operation in the trace tree. When the span is the root of the trace tree, the ParentSpanId is absent.
 
-# Flags
-The following flags are reported either in a flag set or separate attributes.
+# Options
+The following are the only options defined in B3.
 
-## Sampled
+## Sampling Decision
 
-When the Sampled flag is 1, report this span to the tracing system. When it is 0, do not. When B3 attributes are sent without the Sampled flag, the receiver should make the decision. Once Sampled is set to 0 or 1, the same value should be consistently sent downstream.
+When the sampled decision is accept, report this span to the tracing system. When it is reject, do not. When B3 attributes are sent without a sampled decision, the receiver should make one. Once the sampling decision is made, the same value should be consistently sent downstream.
 
 ### Details
```

Comment:
I'd even remove this header and just let the text continue as it's part of the explanation of Spampling Decision

Reply:
Much clearer!

thanks Adrian!

Reply:
thx moved to the FAQ